src/applications/vaos/project-cheetah/index.jsx

```diff
@@ -51,8 +51,8 @@ export function NewBookingSection({ featureProjectCheetah }) {
           component={SelectDate1Page}
         />
         <Route
-          path={`${match.url}/select-date-2`}
-          component={SelectDate2Page}
+          path={`${match.url}/second-dose-page`}
```

Comment:
I think the url could be something more similar to the title of the page, so like `plan-second-dose` or something like that. The word "page" feels unnecessary to me.

Reply:
done